README.md

```diff
@@ -71,29 +75,27 @@ The hook is configured in `.pre-commit-hooks.yaml` with the following settings:
 
 The Docker image supports the following build arguments:
 
-| Argument | Default | Description |
-|----------|---------|-------------|
-| `VERSION` | `1.0.0` | Scanner version number |
-| `GIT_LEAKS_VERSION` | `8.30.0` | GitLeaks version to install |
-| `GIT_LEAKS_SHA512` | (specified) | SHA-512 checksum for verification |
-| `ROOT` | `/app` | Application root directory |
+| Argument            | Default     | Description                       |
+| ------------------- | ----------- | --------------------------------- |
+| `VERSION`           | `local`     | Scanner version number            |
+| `GIT_LEAKS_VERSION` | `8.30.0`    | GitLeaks version to install       |
+| `GIT_LEAKS_SHA512`  | (specified) | SHA-512 checksum for verification |
 
 ## 🏗️ Architecture
 
 ### Components
 
-1. **gitleaks.sh** - Downloads, verifies, and installs GitLeaks
```

Comment:
This will have to stay to adhere to future-proof architecture when additional CLI tools are added.